modules/product/src/Entity/Product.php

```diff
@@ -337,6 +337,23 @@ public static function baseFieldDefinitions(EntityTypeInterface $entityType) {
         ),
       ));
 
+    $fields['store_id'] = BaseFieldDefinition::create('entity_reference')
+      ->setLabel(t('Store'))
+      ->setDescription(t('Store field for products'))
+      ->setCardinality(1)
```

Comment:
Yes, that is correct. We don't have a product model implemented yet. In a product + product variations system, each variation can belong to only one store, while the product (containing marketing info) can be shared.